.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,164 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  backend-cs-check:
+    name: "Backend php-cs-fixer"
```

Comment:
Name is a little misleading, we are not really fixing the files. Maybe cs-check, as in the line above?

Reply:
Also maybe, it would be better if it was like: `Lint: Backend (php-cs-fixer)`

Reply:
Okay, but keep in mind this will be displayed in the GitHub UI as follows:
> CI / Lint: Backend (php-cs-fixer) (pull_request)
Let me know if that doesn't satisfy your visual requirements ;)